jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java

```diff
@@ -96,6 +99,10 @@ protected void setServiceName(final String serviceName) {
         this.serviceName = serviceName;
     }
 
+    protected void setServiceVersion(final String serviceVersion) {
```

Comment:
```suggestion
    public void setServiceVersion(final String serviceVersion) {
```
The other shouldn't be `protected` as well, fixed through #179